addons/mail/models/discuss/mail_message.py

```diff
@@ -1,12 +1,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields
 from odoo.addons.mail.tools.discuss import Store
 
 
 class MailMessage(models.Model):
     _inherit = "mail.message"
 
+    call_channel_id = fields.Many2one("discuss.channel", string="Call Channel", index="btree_not_null")
```

Comment:
I don't think this is necessary, it should use model and res_id instead.

Reply:
If I am not wrong then we would need to check the content of the message body to check whether the message is of Call start/end log or not.